### ISSUE TITLE
Docker 1.3.1 Won't startup up on Scientific Linux 6.5 running kernel-2.6.32-504.1.3.el6.x86_64

### ISSUE THREAD

**Issue by andrewwatson** [docker/docker#9310]
```
# docker version
Client version: 1.3.1
Client API version: 1.15
Go version (client): go1.3.3
Git commit (client): c78088f/1.3.1
```
from /var/log/docker
```
[info] WARNING: You are running linux kernel version 2.6.32-504.1.3.el6.x86_64, which might be unstable running docker. Please upgrade your kernel to 3.8.0.
[info] Listening for HTTP on unix (/var/run/docker.sock)
fatal error: unexpected signal during runtime execution
```
I just updated this box to 2.6.32.504, it was running a slightly older version and had no problems starting up but just a couple of days ago the docker daemon randomly died.  

Has anyone else seen this behavior?

**Comment by jfrazelle** [comment]
pretty sure we don't support this kernel on scientific linux ping @vbatts

**Comment by jfrazelle** [comment]
or @unclejack

**Comment by unclejack** [comment]
@jfrazelle is right.

The workaround being discussed above isn't supported by anyone - it's not supported for the use of Docker by the Docker maintainers and it's not supported by those providing support for RHEL 6/CentOS 6/Scientific Linux 6. This means anyone running any other custom kernel and custom libraries (devicemapper or other libraries on the system) are running their own custom system which is unsupported.

I've run a dynamically linked Docker binary on CentOS 6 with the latest updates and haven't run into any kind of problems. Please make sure to run `yum update` and reboot to update your system.

@jfrazelle I think this can be closed because it's very likely it's been fixed already. Quite a few revisions of the 2.6.32-504 kernel have been released and that makes the 2.6.32-431 kernel really old.
